src/components/Home/Home.tsx

```diff
@@ -1,10 +1,24 @@
-import { Layout, Page, TextStyle, Banner } from "@shopify/polaris";
+import {
+  Layout,
+  Page,
+  TextStyle,
+  Banner,
+  Form,
+  FormLayout,
+  TextField,
+  Button,
+  Card,
+} from "@shopify/polaris";
 import React, { useState } from "react";
 import { PharmacyList } from "../PharmacyList";
 import { TwitterTimelineEmbed } from "react-twitter-embed";
 
 export function Home() {
   const [showBanner, setShowBanner] = useState(true);
+  const [showPostalPrompt, setShowPostalPrompt] = useState(true);
+  const [showInvalidPostal, setShowInvalidPostal] = useState(false);
+  const [renderCards, setRenderCards] = useState(false);
```

Comment:
Would `shouldRenderPharmacyList` would be more precise? "cards" has no semantic meaning.